Test2/Test2_2/Task2/Main.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+#include "Graph.h"
+
+#define ERROR -1
+
+int main(void)
+{
+    bool incidenceMatrices[COUNT_EDGES][COUNT_NODES] =
+    { {1, 1, 0, 0, 0, 0}, {1, 0, 0, 0, 0, 1}, {0, 1, 0, 0, 0, 1}, {0, 0, 1, 1, 0, 0} };
```

Comment:
```suggestion
    bool incidenceMatrices[COUNT_EDGES][COUNT_NODES] =
        { {1, 1, 0, 0, 0, 0}, 
          {1, 0, 0, 0, 0, 1}, 
          {0, 1, 0, 0, 0, 1}, 
          {0, 0, 1, 1, 0, 0} };
```
Ну, чтобы оно визуально было похоже на матрицу